bitnami/airflow/templates/config/configmap.yaml

```diff
@@ -149,6 +149,18 @@ data:
             - secretRef:
                 name: {{ .Values.worker.extraEnvVarsSecret }}
             {{- end }}
+            {{- if .Values.extraEnvVarsSecrets }}
+            {{- range .Values.extraEnvVarsSecrets }}
+            - secretRef:
+                name: {{ .name }}
+            {{- end }}
+            {{- end }}
+            {{- if .Values.worker.extraEnvVarsSecrets }}
+            {{- range .Values.worker.extraEnvVarsSecrets }}
+            - secretRef:
+                name: {{ .name }}
```

Comment:
I don't think we need to add a field `name`. I'd rather use a list with the names.

to instead of `{{ .name }}` it would be ` {{. }}`

Reply:
Ah perfect, this is great learning info for Helm, thanks!

Reply:
So let me make sure I'm formatting this as expected. This worked in testing so I'm just ensuring it's what was intended: 

Current state with `name:`:
`values.yaml`
```yaml
extraEnvVarsSecrets:
  - name: secret-name
```
`configmap.yaml`:
```yaml
{{- range .Values.worker.extraEnvVarsSecrets }}
            - secretRef:
                name: {{ .name }}
```

Desired state: 
`values.yaml`
```yaml
extraEnvVarsSecrets:
  - secret-name
```
`configmap.yaml`:
```yaml
{{- range .Values.worker.extraEnvVarsSecrets }}
            - secretRef:
                name: {{ . }}
```

This produces the following which is what we want: 
```yaml
envFrom:
  - secretRef:
      - name: secret-name
```